### PR TITLE
Qt: Support versions < 5.6

### PR DESCRIPTION
**Pull request by spycrab**

### CODE REVIEW

**Comment by spycrab** [comment]
@dolphin-emu-bot rebuild

File: Source/Core/CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,4 @@ add_subdirectory(UICommon)
 add_subdirectory(VideoCommon)
 add_subdirectory(VideoBackends)
 
-if(ENABLE_QT2)
-  add_subdirectory(DolphinQt2)
-endif()
+add_subdirectory(DolphinQt2)
```
Comment (ligfx) [inline Source/Core/CMakeLists.txt]: Was this meant to be made unconditional?

Comment (spycrab) [reply Source/Core/CMakeLists.txt]: Yes, as you can hopefully see it clearly states '**** TEMPORARY HACK ***' I do not intend to keep this commit around.

**Comment by spycrab** [comment]
For the record: All buildbots have passed when forced to compile DolphinQt2:
* [pr-ubu-x64](https://buildbot.dolphin-emu.org/#builders/2/builds/432)
* [pr-osx-x64](https://buildbot.dolphin-emu.org/#/builders/34/builds/428)
* [pr-freebsd-x64](https://buildbot.dolphin-emu.org/#/builders/34/builds/428)
* etc. (they passed too, I just didn't bother to copy their links)

**Pull request merged**